pkgs/development/python-modules/mastodon/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python-dateutil
+, pytz
+, requests
+, blurhash
+, decorator
+, python_magic
+#, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "mastodon";
```

Comment:
Is the name mastodon.py?

Reply:
I don't know what's the name to keep, it's Mastodon.py [on Pypi](https://pypi.org/project/Mastodon.py/) but the import module is `mastodon`. I've settled to `mastodonpy` for now, but let me know if another name is better.